tests/unit/detection/test_detection_model.py

```diff
@@ -69,71 +78,133 @@ def test_detection_learner_init_model(od_detection_dataset):
 
 
 @pytest.mark.gpu
-def test_detection_learner_train_one_epoch(od_detection_learner):
```

Comment:
would it make sense to split this up between detection and mask? i think it might be cleaner (but more tests)

Reply:
Done